src/instance.js

```diff
@@ -118,6 +129,101 @@ Instance.prototype.create = function(options, callback) {
   this.bigtable.createInstance(this.name, options, callback);
 };
 
+/**
+ * Create an app profile.
+ *
+ * @param {string} name The name to be used when referring to the new
+ *     app profile within its instance.
+ * @param {object} options AppProfile creation options.
+ * @param {'any'|Cluster} options.routing  The routing policy for all
+ *     read/write requests which use this app profile. This can be either the
+ *     string 'any' or an instance of a cluster. This value is required when
+ *     creating the app profile and optional when setting the metadata.
+ * @param {object} [options.gaxOptions]  Request configuration options, outlined
+ *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+ * @param {boolean} [options.allowTransactionalWrites] Whether or not
+ *     CheckAndMutateRow and ReadModifyWriteRow requests are allowed by this
+ *     app profile. It is unsafe to send these requests to the same
+ *     table/row/column in multiple clusters. This is only used when the
+ *     routing value is a cluster.
+ * @param {string} [options.description] The long form description of the use
+ *     case for this AppProfile.
+ * @param {string} [options.etag] Strongly validated etag for optimistic
```

Comment:
Can we remove options.etag for now, since it's not implemented on the server-side?

Reply:
Done